spec/factories/competitions.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :competition do
+    code_name 'Code name'
+    start_date '2015-09-19'
```

Comment:
`start_date` and `end_date` should be dates rather than strings. Faker actually has a nice feature to randomize this.